packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -453,7 +457,7 @@ class CupertinoDatePicker extends StatefulWidget {
   }
 }
 
-typedef _ColumnBuilder = Widget Function(double offAxisFraction, TransitionBuilder itemPositioningBuilder);
+typedef _ColumnBuilder = Widget Function(double offAxisFraction, TransitionBuilder itemPositioningBuilder, Widget magnifier);
```

Comment:
You missed these names here (magnifier -> selectionOverlay)

Reply:
Yes, thanks for reminding me.